source/extensions/filters/listener/proxy_protocol/proxy_protocol.cc

```diff
@@ -42,23 +42,23 @@ Network::FilterStatus Filter::onAccept(Network::ListenerFilterCallbacks& cb) {
 }
 
 void Filter::onRead() {
-  try {
-    onReadWorker();
-  } catch (const EnvoyException& ee) {
-    config_->stats_.downstream_cx_proxy_proto_error_.inc();
-    cb_->continueFilterChain(false);
-  }
+    ProxyState status = onReadWorker();
+
+    if(state == ProxyState::ProtocolFail){
```

Comment:
stats -> status?

Reply:
I will make sure to run the test before committing!

Reply:
I tried running the bazel tests however the git bash returns: "bash: bazel: command not found" Thanks for your help always.

Reply:
You need to follow the build instructions to develop envoy https://github.com/envoyproxy/envoy/blob/master/bazel/README.md

If you don't want to, you can always use docker. https://github.com/envoyproxy/envoy/tree/master/ci#building-and-running-tests-as-a-developer

Reply:
Thank you so much for your help!! @asraa  You are helping me learn the open source contribution so much!